presto-main/src/test/java/com/facebook/presto/block/TestArrayBlock.java

```diff
@@ -110,6 +112,60 @@ public void testWithArrayBlock()
         assertBlockFilteredPositions(expectedValuesWithNull, blockBuilderWithNull.build(), () -> blockBuilder.newBlockBuilderLike(null), 2, 3, 4, 9, 13, 14);
     }
 
+    @Test
+    public void testSingleValueBlock()
+    {
+        // 1 entry array.
+        long[][] values = createTestArray(50);
+        // ArrayBlock holds onto the offsets from builder. Offsets can contain additional elements.
+        // MapBlock copies the required portion of the offsets into a new array.
```

Comment:
I think you mean ArrayBlock here

Reply:
I will remove the comment. I tried using the createBlockBuilderWithValues first, but it sets expected entries to 100 and singleValueBlock returned a different block. I tried capturing that in the comment, which is more confusing.